android/app/src/main/res/layout/launch_screen.xml

```diff
@@ -1,6 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:orientation="vertical" android:layout_width="match_parent"
-    android:layout_height="match_parent">
-    <ImageView android:layout_width="match_parent" android:layout_height="match_parent" android:src="@drawable/splash_screen" android:scaleType="centerCrop" />
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/helio_primary">
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingHorizontal="100dp"
+        android:layout_centerInParent="true"
+        android:src="@drawable/helio"/>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The ImageView's width and height are set to `wrap_content`, which means the view will only be as big as the content requires. If the drawable `helio` is smaller than the screen size, it might not cover the entire screen, unlike the previous implementation where the image was set to match the parent's dimensions. If you want the image to cover the entire screen, consider changing the layout_width and layout_height to `match_parent` and setting the scaleType to `centerCrop` or `fitXY`.

Also, the `android:paddingHorizontal="100dp"` might cause the image to be smaller than intended. If you want to maintain the padding, ensure that the image scales correctly within the padded area.

```diff
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:paddingHorizontal="100dp"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:scaleType="centerCrop"
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->